app/src/main/java/com/example/cs65_final_project/FirebaseDatabaseHelper.java

```diff
@@ -28,21 +28,19 @@
      * Adds an ingredient to Firebase, incrementing the amount of the ingredient if it already exists
      * @param context
      * @param name
-     * @param amount
      */
-    public static void addIngredient(Context context, String name, float amount, String aisle) {
+    public static void addIngredient(Context context, String name, String aisle) {
         DatabaseReference ref = FirebaseDatabase.getInstance().getReference();
         FirebaseAuth auth = FirebaseAuth.getInstance();
         ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).addListenerForSingleValueEvent(new ValueEventListener() {
             @Override
             public void onDataChange(@NonNull DataSnapshot snapshot) {
                 if (snapshot.exists()) { // Ingredient already exists in the database
-                    float existingAmount = snapshot.getValue(Float.class);
-                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).setValue(existingAmount+amount);
+                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name);
                     Log.d("RecipeFinderDatabase", "Added to Existing Ingredient");
                     ((AppCompatActivity)(context)).finish();
                 } else { // Ingredient does not currently exist in the database
-                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).setValue(amount);
+                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).setValue("0");
```

Comment:
Can we just get rid of the value here?

Reply:
Thought I could make the amount an optional part, and just decided to put 0 as a placeholder